sdk/apis/tenancy/v1alpha1/types_workspace.go

```diff
@@ -117,7 +117,7 @@ const LogicalClusterTypeAnnotationKey = "internal.tenancy.kcp.io/type"
 // +kubebuilder:resource:scope=Cluster,categories=kcp,shortName=ws
 // +kubebuilder:printcolumn:name="Type",type=string,JSONPath=`.spec.type.name`,description="Type of the workspace"
 // +kubebuilder:printcolumn:name="Region",type=string,JSONPath=`.metadata.labels['region']`,description="The region this workspace is in"
-// +kubebuilder:printcolumn:name="Phase",type=string,JSONPath=`.metadata.labels['tenancy\.kcp\.io/phase']`,description="The current phase (e.g. Scheduling, Initializing, Ready, Deleting)"
+// +kubebuilder:printcolumn:name="Phase",type=string,JSONPath=`.status.phase`,description="The current phase (e.g. Scheduling, Initializing, Ready, Deleting)"
```

Comment:
is this a no-op? Is the code removed for that label? Is the label removed?

Reply:
I dont understand why label was used in the first place. Mounts dont have this label. 
Why label was used in the first place but not status itself as reference for column?

Reply:
Maybe there are states not represented in the phase?

Reply:
No idea. I was not able to find a case of why this was done. Just looks like some side conversations which was not documented 🤷 
It was introduced in bootstrapping PR: https://github.com/kcp-dev/kcp/pull/2595